src/style.css

```diff
@@ -113,75 +113,73 @@ html, body {
 }
 
 .fullscreen {
-  width:100%;
-  height:100vh;
-  margin:0;
-  padding:0;
+  width: 100%;
+  height: 100vh;
+  margin: 0;
+  padding: 0;
 }
 
 .fill_space {
-  width:100%;
-  height:100%;
+  width: 100%;
+  height: 100%;
   overflow: auto;
   min-height: 0
 }
 
-/* landing page */
-
+/* Landing page */
 #landing_page {
-  position:relative
+  position: relative
 }
 
 #landing_page_content {
-  position:absolute;
-  top:50%;
-  left:50%;
+  position: absolute;
+  top: 50%;
+  left: 50%;
   transform: translate(-50%, -50%);
 }
 
 #landing_page_content h1 {
-  font-size:20vmin;
-  margin:0;
-  padding:0;
+  font-size: 20vmin;
+  margin: 0;
+  padding: 0;
   transition: text-shadow 1s ease-in-out 1s;
 }
 
 #landing_page_content h1:hover {
   text-shadow:
-            +0px   0px   15px   var(--bg-color),
-            +0px   -15px   20px   rgba(255,0,0,0.5),
-           -15px   +10px   20px   rgba(0,255,0,0.5),
-           +10px   +15px   20px   rgba(0,0,255,0.5);
+							 +0px     0px   15px   var(--bg-color),
+						   +0px   -15px   20px   rgba(255,0,0,0.5),
+							-15px   +10px   20px   rgba(0,255,0,0.5),
+							+10px   +15px   20px   rgba(0,0,255,0.5);
```

Comment:
Could you reverse this change? It looks like you used indentation.

Reply:
I indented it like that so the px is on the same column but I'll fix it in a sec 👍

Reply:
Also shouldn't this have some transition property because it's a :hover element? I have to hover for a few seconds before it shows up, very cool effect though.

Reply:
The transition is:
```
  transition: text-shadow 1s ease-in-out 1s;
```

So, we wait for 1 second before triggering the transition. This was done on purpose to make this a bit harder to find. This is a sort of easter eggs.